install

```diff
@@ -125,8 +115,7 @@ do_install()
   rosinstall_generator ${ROS_CONFIGURATION} ${ROS_EXTRA_PACKAGES} --rosdistro ${ROS_DISTRO} --deps --tar > ${WS}.rosinstall
   wstool init -j8 src ${WS}.rosinstall
 
-  # Grabbing these older meshes allows rviz to run with Ogre 1.7 rather than Ogre 1.8+. Although formulae and patches
-  # exist to build Ogre 1.9, rviz, and Gazebo together, I have found this approach to be the most solid and consistent.
+  # Grabbing these older meshes allows rviz to run with Ogre 1.7 rather than Ogre 1.8+.
   if [ -d src/rviz ]; then
     pushd src/rviz/ogre_media/models
     curl https://raw.githubusercontent.com/ros-visualization/rviz/hydro-devel/ogre_media/models/rviz_cone.mesh > rviz_cone.mesh
```

Comment:
@wjwwood What's the chance that rviz could include these meshes for multiple Ogre versions, and use a compile-time switch to select between them? It would be a lot more elegant than me doing this awfulness here.